.github/workflows/benchmark.yml

```diff
@@ -62,6 +62,8 @@ jobs:
       run: BIG=2 MPS=1 python3.11 test/test_speed_v_torch.py | tee torch_speed.txt
     - name: Test tensor cores
       run: METAL=1 python3.11 test/test_linearizer.py TestLinearizer.test_tensor_cores TestLinearizer.test_tensor_cores_padded
+    - name: Test AMX tensor cores
+      run: CLANG=1 AMX=1 python3.11 test/test_linearizer.py TestLinearizer.test_tensor_cores TestLinearizer.test_tensor_cores_padded
```

Comment:
you would need `DEBUG=2` or some other ways to tell if AMX is really being used

Reply:
Sorry, I thought the tests asserted that they were triggered

Reply:
oh yea it asserts, i was thinking running `simple_matmul` with AMX like other gemm